test/generator/exercise_test.rb

```diff
@@ -95,9 +95,9 @@ def test_bookkeeping
       Object.send(:remove_const, :AlphaCase)
     end
 
-    def test_exercise_name
+    def test_name
       subject = Exercise.new(paths: FixturePaths, slug: 'alpha-beta')
-      assert_equal 'alpha_beta', subject.exercise_name
+      assert_equal 'alpha_beta', subject.name
```

Comment:
I REALLY don't like this being a part of `Exercise` now.
(but yeah, refactor later.)